src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala

```diff
@@ -20,9 +20,29 @@ import com.spotify.missinglink.datamodel.{
 }
 
 object MissingLinkPlugin extends AutoPlugin {
-  object autoImport {
+
+  trait Types {
+    final case class IgnoredPackage(name: String, ignoreSubpackages: Boolean = true)
```

Comment:
Please put this directly inside `autoImport`. There is no need to create a path-dependent case class inside a `trait Types`.